dora/core/client/fs/src/main/java/alluxio/client/file/DoraCacheFileSystem.java

```diff
@@ -137,7 +137,7 @@ public URIStatus getStatus(AlluxioURI path, GetStatusPOptions options)
     }
     try {
       GetStatusPOptions mergedOptions = FileSystemOptionsUtils.getStatusDefaults(
-          mFsContext.getPathConf(path)).toBuilder().mergeFrom(options).build();
+          mFsContext.getClusterConf()).toBuilder().mergeFrom(options).build();
```

Comment:
I am not quite sure if simply replacing `getPathConf` by `getClusterConf` (a pattern in this PR) is always correct.
Can you double-check it?

Reply:
With the recent change, there is no configure propagation, everything depends on the local config, there should be no difference.